README.md

```diff
@@ -1,2 +1,157 @@
 # time-monkey
 Base application for management of employees working hours
+
+## Environment set-up
+
+### Ubuntu
+
+**This project uses Python in version 3.7.0. If you're already using the same or higher version, you can skip the following step.**
+
+1. Install Python by running in Terminal 'sudo apt install python3.7'. You can verify the installation by running 'python3.7 --version'
+
+**Skip this step if PIP is already installed.**
+
+2. Install PIP with `sudo apt install python3-pip`.  You can verify the installation by running `python3.7 -m pip --version`.
+
+3. If you don't have virtualenv installed, run installation with PIP by running `sudo python3.7 -m pip install virtualenv` in Terminal
+
+4. In Terminal, navigate to directory you wish to install virtual environment to and run `python3.7 -m virtualenv <name for your venv>`.
+    You can now activate your virtual environment by running `. <your venv location>/bin/activate`. To deactivate it simply type `deactivate` in Terminal.
+
+5. To install PostgreSQL run `sudo apt install postgresql postgresql-contrib` in Terminal.
+
+6. Edit pg_hba.conf file located in */etc/postgresql/<version>/main/*. Edit the following line:
+    ```
+    local   all             postgres                                <method>
+    ```
+    Change <method> to trust.
+
+7. Start PostgreSQL service with `sudo service postgresql start`. You can also use commands ***restart*** and ***stop*** to control the service.
+
+8. Create database for application by running the following command in Terminal: `createdb -U postgres time_monkey`. If you get ***peer authentication failed*** message, restart PostgreSQL service.
+
+9. Navigate with Terminal to where you want the repository to be cloned and run `git clone https://github.com/Code-Poets/time-monkey.git`
```

Comment:
But like you said, you never know who might show up for the job in the future! Primary schools are already doing coding, you know... xD

Reply:
Actually, I will have to move this step somewhere, since you described requirements installation back at the virtualenv.
Is it relevant where I move it? Shall I put this before virtualenv or maybe at the very beginning?

Reply:
Done